ArduCopter/mode.h

```diff
@@ -872,7 +872,7 @@ class ModeGuided : public Mode {
 
     bool requires_terrain_failsafe() const override { return true; }
 
-    void set_angle(const Quaternion &q, float climb_rate_cms_or_thrust, bool use_yaw_rate, float yaw_rate_rads, bool use_thrust);
+    void set_angle(const Quaternion attitude_quat, const Vector3f &ang_vel, float climb_rate_cms_or_thrust, bool use_thrust);
```

Comment:
We should have a comment above this method and/or in the .cpp file clarifying that attitude_quat should be zero if only an angular velocity is provided.

Also the "const Quaternion attitude_quat" can be passed by reference.  So it should be, "const Quaternion& attitude_quat"

Reply:
I don't think this is quite right. 

This new scheme allows you to specify both the attitude and rates at the same time. If the attitude quaternion is zero then we use only the angular velocity instead through a different function.

I've added a new comment to clear this up hoepfully?

Reply:
great, the comment looks very clear now, txs!